test/test_c10d.py

```diff
@@ -499,6 +500,42 @@ def is_master(self):
         return self.rank == 0
 
 
+class TimeoutTest(TestCase):
+    def _test_store_timeout(self, backend, init_method, c2p):
+        c10d.distributed_c10d.init_process_group(
+            backend=backend, init_method=init_method, world_size=1, rank=0,
+            timeout=timedelta(seconds=1))
```

Comment:
@pietern I found that when using `FileStore`, the elapsed time is a little larger than 2 seconds instead of 1 second (tried only twice), but the code in `FileStore` only sleeps 10 milliseconds between two iteration. Is this an expected behavior?

Reply:
This is odd. I see you "fixed" it by using the 3s threshold which is reasonable I guess.